src/routes/participants/+page.server.ts

```diff
@@ -13,6 +13,7 @@ export const load: PageServerLoad = async (): Promise<{ participants: Participan
 		const participantsFilePaths = await _loadParticipantJsonFilePaths(PARTICIPANTS_DIRECTORY);
 
 		const participants = await _loadParticipants(participantsFilePaths);
+		participants.sort((a, b) => (a.name < b.name ? -1 : a.name === b.name ? 0 : 1));
```

Comment:
Shouldn't be simple `.sort()` enough?
```suggestion
		participants.sort();
```

Reply:
Ah never mind, I got confused.

Reply:
Ok 😅 I tried it myself without the compare function and got confused as well - the first few participants are already sorted, but for example I would still not be sorted correctly. We need to check against properties inside the object.